config/locales/user_mailer/en.yml

```diff
@@ -1,34 +1,34 @@
+---
 en:
   user_mailer:
     contact_request:
-      comments_header: "Comments:"
       comments: No comments left
-      email_or_phone: "Email address or phone number:"
-      'no': no
+      comments_header: 'Comments:'
+      email_or_phone: 'Email address or phone number:'
+      'no': false
       talk_about_experience: Talk about experience?
       want_to_learn: Learn more?
-      'yes': yes
+      'yes': true
```

Comment:
Yeah I agree with that task, we were wrong to use an unquote `yes` value, it should have been `"yes"`